pxr/imaging/hgi/types.cpp

```diff
@@ -246,14 +246,18 @@ HgiIsFloatFormat(
     case HgiFormatSNorm8:
     case HgiFormatFloat16:
     case HgiFormatFloat32:
-    case HgiFormatFloat32UInt8: // Unclear
     case HgiFormatBC6FloatVec3:
     case HgiFormatBC6UFloatVec3:
     case HgiFormatPackedInt1010102:
         return true;
     case HgiFormatInt16:
     case HgiFormatUInt16:
     case HgiFormatInt32:
+    // While this looks like a float with packed integer format,
```

Comment:
I wonder if this is an opportune moment to name this what it is, which is HgiFormatDepth32UInt8. The mapping from Hgi to graphics APIs would be far less ambiguous.

Reply:
I could, but it's 19 usages in 10 new files not in the PR. Also I would recommend naming it `HgiFormatDepthFloat32Stencil8` since some APIs like Metal and Vulkan distinguish between colour and depth floats (`VK_FORMAT_D32_SFLOAT` vs `VK_FORMAT_R32_SFLOAT`, `MTLPixelFormatDepth32Float` vs `MTLPixelFormatR32Float`), and stencil is a bit mask not an integer.

Reply:
Since you called it out in the comment, it seemed worth asking. I do agree that not spilling out into more files is preferable.